### DISCUSSION TITLE
Status text issue

### DISCUSSION THREAD

**Discussion by murh2081** [community/community#116839] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The status text is not displayed when hovering over status icon while viewing your own profile.

**Comment by Chamindu77** [comment]


**Reply by Akash1134** [reply]
Hi @Chamindu77 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
